New UI design for profiles page added

Hyper-Matrix opened the pull request:
### Description
The new UI for My Profile screen has been added (mockup)
Fixes #807 

### Type of Change:




- User Interface


**Code/Quality Assurance Only**

- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
GIF- 
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/48028414/91973477-f5254280-ed39-11ea-9425-e14e7ed3b504.gif)

ScreenShot-
![profile](https://user-images.githubusercontent.com/48028414/91973497-fe161400-ed39-11ea-838b-4f02b2198bf6.jpg)


### Checklist:


- [x] My PR follows the style guidelines of this project



**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

Hyper-Matrix committed:
Create render.experimental.xml

Hyper-Matrix committed:
Merge branch 'develop' of https://github.com/Hyper-Matrix/mentorship-android into develop

Hyper-Matrix committed:
Edited request activity

Added alert for accept , reject and delete requests.

Hyper-Matrix committed:
Edited feedback activity

Added alert dialogue to feedback section

Hyper-Matrix committed:
Create ui.txt

Hyper-Matrix committed:
profile activity UI

vj-codes reviewed (changes requested):
@Hyper-Matrix you were supposed to submit  ```In one of png / jpeg / Figma URL / ai formats.
 In the PR add an image of the design to /docs/ui folder. ``` and not code the design

Hyper-Matrix commented:
@vj-codes I have added the jpeg files in the docs/UI folder. I haven't codded anything, the code is for issues #824 #826. So I created a branch UI and added it. If you can see I have created a folder doc and UI and added the image. If am wrong anywhere please guide me out.

iofall commented:
@Hyper-Matrix you need to separate your commits across issues. This issue is only for redesign so it should only contain commits for redesign and not any other issue. Please remove the other commits and only keep those that are relevant for this issue.

iofall reviewed (changes requested):
For each issue, you should create a separate branch which helps in separation of commits across issues.

iofall commented:
Also @Hyper-Matrix please remember to follow the [Contributing Guidelines](https://github.com/anitab-org/mentorship-android/blob/develop/.github/CONTRIBUTING.md) your commit messages are not according to the guidelines.

Hyper-Matrix commented:
@iofall I have created a a separate branch for all(Like for this issue I created a branch UI) . But since my other PRs have not been merged changes reflect in every new PR even after I create a new branch. Is there a way to avoid this?

Hyper-Matrix committed:
Delete FeedbackActivity.kt

Hyper-Matrix commented:
I closing this PR and submitting a new one.
